react/src/components/GeneSearch.tsx

```diff
@@ -43,6 +63,22 @@ const GeneSearch: React.FC<GeneSearchProps> = ({ assembly, geneName, onChange, o
 
     const debouncedAutocompleteFetch = useAsyncDebounce(fetchAutocompleteResults, 500);
 
+    const formatAutocompleteOptions = useCallback(
+        (autocompleteResults: AutocompleteResults) =>
+            (autocompleteResults.autocompleteResults.hits || [])
+                .filter(hit => !!hit.ensembl?.gene && !!hit.genomic_pos && !!hit.genomic_pos_hg19)
```

Comment:
Guessing by this that most fields are actually optional for `AutocompleteResults`?

Reply:
ah yes good catch.